utils/runtime-fuzzer/src/runtime/account.rs

```diff
@@ -106,10 +106,22 @@ impl<'a> BalanceManager<'a> {
 
     pub(crate) fn update_balance(&mut self) -> Result<BalanceState> {
         let max_balance = runtime::gas_to_value(runtime::acc_max_balance_gas());
-        let new_balance = self
-            .unstructured
-            .int_in_range(EXISTENTIAL_DEPOSIT..=max_balance)?;
 
+        // In 1/4 cases we're going to get max_balance account which helps us to run code to completion.
+        //
+        // Note that before there was another branch here that also did more calculation on `max_balance` to get into the sweet spot
+        // but it turns out to slightly move the balance of success/failure rate to 50/50 which is not good. With only these two branches
+        // we get around 80/20 success/failure rate. Note that this also depends on number of instructions in the program.
```

Comment:
what eventually affected such adjustment from 50\50 to 80\20?

Reply:
Before it was essentially `max_balance.saturating_div(4).max(EXISTENTIAL_DEPOSIT)` and its variations I tried, this did not result in anything useful so I've just removed that branch. It's much better to either give max balance to program or give it random balance up to max balance from countless run observations.